synapse/lib/cell.py

```diff
@@ -2151,3 +2158,30 @@ async def kill(self, user, iden):
         perm = '.'.join(perm)
         raise s_exc.AuthDeny(mesg=f'User must have permission {perm} or own the task',
                              task=iden, user=str(user), perm=perm)
+
+    async def getCellVersionInfo(self):
+        '''
+        Return version metadata specific for this Cell.
+
+        Notes:
+            By default, this function returns information about the base Cell
+            implementation, which relects the base information in the Synapse
+            Cell.
+
+            It is expected that implementers override the following Class
+            attributes in order to provide meaningful version information:
+
+            ``COMMIT``  - A Git Commit
+            ``VERSION`` - A Version tuple.
+            ``VERSTRING`` - A Version string.
+
+        Returns:
+            Dict: A Dictionary of version metadata.
+        '''
+        ret = {
+            'commit': self.COMMIT,
```

Comment:
tempted to add ``name`` here too?

Reply:
We should probably bake the synapse version information in as well

Reply:
Name and synapse version info seems like a good addition, especially if we're going with naming the API `getCellInfo`.

Reply:
:+1:

Reply:
@vEpiphyte if your "adding `name`" comment above is about the type of cell, i completely agree, but lets add it as "type" since that makes more sense from a "sorting various contents" perspective